kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -139,6 +142,24 @@
                         ++orderInGroup,
                         Width.LONG,
                         "S3 accelerated endpoint enabled");
+
+      CONFIG_DEF.define(S3_PROXY_HOST,
+              Type.STRING,
+              Importance.HIGH,
+              "proxy host.",
```

Comment:
These should be straightforward to understand, but perhaps we could get a full sentence description for the documentation?

nit: Fixing the capitalization would be nice too :)